test/interval_tests/parsing.jl

```diff
@@ -1,67 +0,0 @@
-using IntervalArithmetic
```

Comment:
I notice that you have removed the parsing tests (this file). While I a-priori might agree, the question is: are all instances covered by other (ITF1788) tests? Maybe these tests should be simply re-adapted to test other numeric formats, e.g. `Float32`.

Reply:
in principle ITF1788 can generate tests for other number format (the original python script not mine). Directly using the testsuite for other number format probably will not work (double rounding for Float32, BigFloat would probably be tighter). 

I heard rumors there's some work for a unified testing framework ;) meanwhile, I'll reinsert some manual tests for other formats

Reply:
While you may be right with respect to ITF1788, I don't think "other formats" includes extended precision, simply because the "native" way of coding those tests is `Float64`. In any case, reinserting those tests with another format was my point